lib.php

```diff
@@ -37,11 +46,9 @@
  * See also https://docs.moodle.org/dev/Login_callbacks#after_config.
  */
 function tool_excimer_after_config(): void {
-    // Skip setup if unit testing, but do the test here to avoid complicating manager::is_profiling().
-    if (!PHPUNIT_TEST && manager::is_profiling()) {
-        $manager = manager::create();
-        $manager->init();
-    }
+    // Start processor.
+    $manager = manager::get_instance();
+    $manager->start_processor();
```

Comment:
Why is the logic for what _session_start is doing different to what _after config is doing? I would assume they should be identical

Reply:
This logic had to be separated out to start the timer 'tool_excimer_before_session_start' and then flesh the details in `tool_excimer_after_config` as explained here https://github.com/catalyst/moodle-tool_excimer/pull/262#issuecomment-1237780878